### ISSUE TITLE
Add option to download later Wine builds (eg. from Kron4ek)

### ISSUE THREAD

**Issue by Snaggly** [Heroic-Games-Launcher/HeroicGamesLauncher#4001]
### Problem description

Certain games on GOG such as "The Settlers II: 10th Anniversary" have resolution/framerate bugs when run on latest GE-Wine, or "Monster Boy and the cursed Kingdom" flat out crashes at start which have been already long fixed with more up to date Wine builds.

### Feature description

Having other Wine builds, such as Kron4ek's, to be downloadable would be pretty nice as well. Current situation is that you can only either download GE-Wine or GE-Proton, or use the system installed Wine (which on a stock Steam Deck is not a nice option) or the Proton installs from Steam.

GE-Wine is pretty much dead at this point, leaving out many bugfixes and advancements that came with more later Wine builds and GE-Proton as well as the later official Proton builds from Valve don't work with games that aren't offered on the Steam store. I would get this on the log if I tried running "The Settlers II: 10th Anniversary" from GOG using GE-Proton:
>S_API FAIL] SteamAPI_Init() failed; no appID found.
> 
> Either launch the game from Steam, or put the file steam_appid.txt containing the correct appID in your game folder.
> 
> wine: using kernel write watches, use_kernel_writewatch 1.
> 
> Launch command: ['/home/snaggly/.local/share/Steam/compatibilitytools.d/GE-Proton9-10/proton', 'waitforexitandrun', > '/home/snaggly/Games/Heroic/The Settlers II - 10th Anniversary/bin/S2DNG.exe', '-fullscreen']
> 
> All processes exited
> 
> ============= End of log =============

### Alternatives

At this point you can only download other wine builds besides GE-Wine to Heroic using ProtonPlus, this would require a newbie to know about an existence of such a tool as he would most likely just download the latest what  is available on the platform and then get easily frustrated that this and that game does not work with the offerings inside this bubble.

### Additional information

In the past, there has been some effort to accomplish something similar: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/2024
Although only integrating the latest Bottle's Soda builds. The PR was left stale for almost 2 years now.

**Comment by mbtns** [comment]
Indeed, from my end I need wine-ge rather than the proton version predominantly because I use a DualSense controller which is - for some reason - not recognized when using the proton-ge version, but is when using the wine one. So I feel there still is a need for these types of prefixes (also because the add to steam function does not allow me to launch the game). 

I am aware that I can add these wine versions manually via [the tools folder](https://github-wiki-see.page/m/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/How-To%3A-Wine-and-Proton) - as the add custom wine/proton does not appear to work for me at this moment. 

I am on Fedora Silverblue 40 and use heroic and steam as flatpaks.

**Comment by casasfernando** [comment]
I second this request.
I have old retail games in my library which are not compatible or doesn't work with Wine-GE or Proton-GE (e.g.: Grand Prix 4, Half-Life -WON-) but they work flawlessly with Kron4ek's vanilla Wine builds (stable or staging).
These builds are compatible with Steam Deck as well.
It would be great to be able to get them directly from Heroic's UI instead of having to manually download, extract and add them to Heroic through the tools folder or the custom Wine option.

Thanks
